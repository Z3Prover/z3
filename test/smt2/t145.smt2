
; Copyright (c) 2015 Microsoft Corporation
(declare-const v0 (_ BitVec 8))
(simplify (bvashr (bvashr v0 #x01) v0))
(declare-const v1 (_ BitVec 8))
(simplify (bvashr (bvashr v0 v1) #x01))