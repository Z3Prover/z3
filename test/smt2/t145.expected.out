(bvashr (bvashr v0 #x01) v0)
(bvashr (bvashr v0 v1) #x01)
